### ISSUE TITLE
VC video panels can sometimes disappear on some clients when participants join

### ISSUE THREAD

**Issue by rxl881** [matrix-org/matrix-vr-demo#21]
We haven't been able to capture logs of this happening yet. So, creating this as a placeholder for now.

**Comment by superdump** [comment]
If I understood correctly - the symptom is that with a Firefox in an ongoing conference, sometimes when some other client joins the VC, all videos can disappear.

**Comment by ara4n** [comment]
I /think/ i've seen this with pure Chrome too - where all the clients may end up out of sync with different combinations of full mesh calls running.

**Comment by ara4n** [comment]
Marking this as a release-blocker to encourage us to try to reproduce it in Chrome and if we can't, close it.

**Comment by superdump** [comment]
Random thought about why this could happen - some error at the beginning of the render function for the ConferenceView? Some error in obtaining the calls and constructing the CallViews?

**Comment by superdump** [comment]
<img width="1680" alt="screen shot 2017-04-18 at 10 12 45" src="https://cloud.githubusercontent.com/assets/302146/25120761/a228a8ba-241f-11e7-8503-0ec9ef2071bd.png">

I managed to reproduce on 3x Chrome Version 57.0.2987.133 (64-bit) on macOS. The top-right initiated and almost immediately I started the top-left and bottom-left. The top-right, as can be seen, has no visible participants but appears to be present in the conference for the other two.
